programs/manifest/src/validation/loaders.rs

```diff
@@ -544,6 +555,14 @@ impl<'a, 'info> BatchUpdateContext<'a, 'info> {
                     let global_fixed: &GlobalFixed = get_helper::<GlobalFixed>(&global_data, 0_u32);
                     let expected_global_vault_address: &Pubkey = global_fixed.get_vault();
 
+                    let global_mint_key: &Pubkey = global_fixed.get_mint();
+                    let (expected_global_key, _global_bump) = get_global_address(global_mint_key);
```

Comment:
why not mint.info.key?

Reply:
that also works. just consistent same check everywhere this way